<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditClusters</class>
 <widget class="QDialog" name="DialogEditClusters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clusters</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 66, 74);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btNew">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/plus-solid.svg</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btRemove">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/trash-solid.svg</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btMoveUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/angle-up-solid.svg</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btMoveDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/angle-down-solid.svg</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listClusters">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbNabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #151515;
border: 0px;
padding: 3px 8px ;
border-radius: 10px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>btNew</tabstop>
  <tabstop>btRemove</tabstop>
  <tabstop>btMoveUp</tabstop>
  <tabstop>btMoveDown</tabstop>
  <tabstop>listClusters</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
